models/Single-Pendulum/Single-Pendulum.jl

```diff
@@ -23,18 +23,17 @@
 
 using NeuralNetworkAnalysis
 
-#models_dir = normpath(@__DIR__, "..", "..", "..", "models")
-#path = joinpath(models_dir, "SinglePendulum", )
-controller = read_nnet(@relpath "controller_single_pendulum.nnet")
+models_dir = normpath(@__DIR__, "..", "..", "..", "models")
+path = joinpath(models_dir, "Single-Pendulum", "controller_single_pendulum.nnet")
+controller = read_nnet(path)
```

Comment:
I still do not understand this. How and why does this work?

Reply:
just `models_dir` moves out of the `docs` folder (where this script is actually run from), and `path` attaches the `.nnet` file. 

idk if there is a cleaner / standard way to approach this, but i'm fine to use this method (the same is used in the Examples of RA when they include model files)

Reply:
But why does it work when I am in the folder `models/Single-Pendulum/`? Then doing `cd ../../../models` would fail. I just do not understand this.

Reply:
`@__DIR__` returns the directory of the file containing the macro call. if i'm not mistaken that's  `GENERATEDDIR`, under `docs/src/models`.

Reply:
As I suspected, this does not work anymore: Go to `models/Single-Pendulum/`, start a Julia session and type:
```julia
julia> include("Single-Pendulum.jl")
ERROR: LoadError: SystemError: opening file "~/.julia/dev/models/Single-Pendulum/controller_single_pendulum.nnet": No such file or directory
```
I would like to let this work rather than the documentation. I can propose a fix later.

Reply:
yes, to run each file as a standalone script we need to fix the path differently. the way i test is to build the docs locally with `$ julia docs/make.jl`, then it finds the file with this PR.